admin/views/tabs/sitemaps/taxonomies.php

```diff
@@ -18,9 +18,16 @@
 if ( is_array( $taxonomies ) && $taxonomies !== array() ) {
 	foreach ( $taxonomies as $tax ) {
 		// Explicitly hide all the core taxonomies we never want in our sitemap.
-		if ( in_array( $tax->name, array( 'link_category', 'nav_menu', 'post_format' ) ) ) {
+		if ( in_array( $tax->name, array( 'link_category', 'nav_menu' ) ) ) {
 			continue;
 		}
+
+		$all_options = WPSEO_Options::get_all();
```

Comment:
It it possible to use WPSEO_Options::get_option() instead of get_all(). It seems to only be used for disable-post-format.